src/main/java/com/github/orest/car_shop/controllers/HomePageController.java

```diff
@@ -0,0 +1,21 @@
+package com.github.orest.car_shop.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+@Controller
+public class HomePageController {
+    @GetMapping("/home")
+    public String home(@RequestParam(name="name", required=false, defaultValue="World") String name, Model model) {
```

Comment:
You don't need to specify `name="name"` for `@RequestParam` because It is already named as `name`